### PR TITLE
[TPC-QC] Add DCAr selection to Tracks.cxx

### PR DESCRIPTION
**Pull request by makor**

### CODE REVIEW

File: Detectors/TPC/qc/src/Tracks.cxx [hunk 1]
```diff
@@ -207,6 +212,46 @@ bool Tracks::processTrack(const o2::tpc::TrackTPC& track)
...
+        mMapHist["h2DEtaPhiPos"]->SetTitle("h2DEtaPhiPos o2::base::Propagator not properly initialized");
+        mMapHist["hEtaVsPtPos"]->SetTitle("hEtaVsPtPos o2::base::Propagator not properly initialized");
+        mMapHist["hPhiVsPtPos"]->SetTitle("hPhiVsPtPos o2::base::Propagator not properly initialized");
+      }
+      return true;
+    }
+
+    if (dcaValue > mCutMaxAbsDCAr) {
+      return true;
+    }
```
Comment (wiechula) [inline Detectors/TPC/qc/src/Tracks.cxx]: Shouldn't this be
```suggestion
    if (mUseCutMaxAbsDCArOnHistos && (dcaValue > mCutMaxAbsDCAr)) {
      return true;
    }
```

**Review by wiechula — CHANGES_REQUESTED** [review]

File: Detectors/TPC/qc/src/Tracks.cxx (hunk 1 above)
Comment (shahor02) [reply Detectors/TPC/qc/src/Tracks.cxx]: actually, the dcaValue is signed, so one should check `std::abs(dcaValue)`

Comment (makor) [reply Detectors/TPC/qc/src/Tracks.cxx]: That is totally true, I will push an update and fix this. Thanks a lot!

**Review by wiechula — APPROVED** [review]

**Comment by makor** [comment]
@shahor02 @davidrohr 

Could you approve this PR? 

Thanks a lot 
Max

**Review by shahor02 — APPROVED** [review]

**Pull request merged**